### PR TITLE
Add shared option to ParaView.

### PR DESCRIPTION
**Pull request by danlipsa**

### CODE REVIEW

**Comment by danlipsa** [comment]
@chuckatkins Please review. I need this for building statically on supercomputers to mitigate file system load for dynamic libraries and python files.

File: var/spack/repos/builtin/packages/paraview/package.py [hunk 1]
```diff
@@ -235,6 +238,15 @@ def nvariant_bool(feature):
                 '-DMPI_Fortran_COMPILER:PATH=%s' % spec['mpi'].mpifc
             ])
 
+        if '+shared' in spec:
+            cmake_args.extend([
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/paraview/package.py]: If you're only adding a single string, you can use `append()` instead of `extend([])`

**Comment by danlipsa** [comment]
@adamjstewart Thanks! I made the change suggested.

File: var/spack/repos/builtin/packages/paraview/package.py [hunk 2]
```diff
@@ -235,6 +238,15 @@ def nvariant_bool(feature):
                 '-DMPI_Fortran_COMPILER:PATH=%s' % spec['mpi'].mpifc
             ])
 
+        if '+shared' in spec:
+            cmake_args.append([
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/paraview/package.py]: Append takes a string, not a list

Comment (danlipsa) [reply var/spack/repos/builtin/packages/paraview/package.py]: Good catch!

**Pull request merged**